grafana-altera-datasource/src/ConfigEditor.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { ComponentType } from 'react';
+import { DataSourcePluginOptionsEditorProps } from '@grafana/data';
+import { DataSourceHttpSettings } from '@grafana/ui';
+
+import { DataSourceOptions } from './types';
+
+type Props = DataSourcePluginOptionsEditorProps<DataSourceOptions>;
+
+export const ConfigEditor: ComponentType<Props> = ({ options, onOptionsChange }) => {
+  return (
+    <>
+      <DataSourceHttpSettings
+        defaultUrl={'http://localhost:8080'}
```

Comment:
đang hard code cái url, bỏ ra 1 file config riêng chứa toàn bộ config như thế này